sketch_map_tool/map_generation/qr_code.py

```diff
@@ -0,0 +1,82 @@
+import json
+from datetime import datetime, timezone
+from io import BytesIO
+from typing import Dict, List
+
+import qrcode
+import qrcode.image.svg
+from reportlab.graphics.shapes import Drawing
+from svglib.svglib import svg2rlg
+
+from sketch_map_tool import __version__
+
+
+def qr_code(
+    uuid: str,
+    bbox: List[float],
+    format_: str,
+    orientation: str,
+    size: Dict[str, int],
+    scale_factor: float,
+    version: str = __version__,
+    timestamp: datetime = datetime.now(timezone.utc),
+) -> Drawing:
+    """Generate a QR code holding the Celery task id and parameters of the map creation.
+
+    Args:
+        uuid: The uuid of a celery task associated with the creation of the PDF map.
+    """
+    data: list = _to_text(
+        uuid,
+        bbox,
+        format_,
+        orientation,
+        size,
+        version,
+        timestamp,
+    )
+    qr_code_svg = _make_qr_code(data)
+    qr_code_rlg = _to_report_lab_graphic(qr_code_svg, scale_factor)
+    return qr_code_rlg
+
+
+def _to_text(
```

Comment:
If we use the types `Bbox` and `PaperFormat`, this function could be removed as you could simply have something like
```
texts = [str(element) for element in data]
```